packages/markdown-docx/package.json

```diff
@@ -73,8 +73,8 @@
     "jsdoc": "3.6.3",
     "jsome": "2.5.0",
     "license-check": "1.1.5",
-    "mocha": "6.1.4",
-    "nyc": "14.1.1",
+    "mocha": "^9.0.0",
+    "nyc": "^15.1.0",
```

Comment:
Didn't it modify the package-lock.json?

Reply:
It should have modified but since it can be recreated using this, I didn't push.

Reply:
`pacakge-lock.json` is file which must be committed. It tells the exact versions of the package unlike `package.json`. For a more elaborate reasoning, refer to [this post](https://stackoverflow.com/a/44297998/11751642).